test/backend_test.in.cpp

```diff
@@ -7422,38 +7437,8 @@ NGRAPH_TEST(${BACKEND_NAME}, avg_pool_3d_strided_uneven_padded_do_not_include_in
     }
 }
 
-NGRAPH_TEST(${BACKEND_NAME}, avg_pool_3d_uneven_strided_padded_include_in_computation)
-{
-    Shape shape_a{64, 3, 7, 8, 10};
-    Shape window_shape{2, 3, 2};
-    auto move_strides = Strides{2, 3, 4};
-    Shape padding_below{5, 6, 4};
-    Shape padding_above{6, 4, 5};
-    auto A = make_shared<op::Parameter>(element::f32, shape_a);
-    auto B = make_shared<op::Parameter>(element::f32, shape_a);
-
-    auto cpu_f = make_shared<Function>(
-        make_shared<op::AvgPool>(A, window_shape, move_strides, padding_below, padding_above, true),
-        op::ParameterVector{A});
-    auto int_f = make_shared<Function>(
-        make_shared<op::AvgPool>(B, window_shape, move_strides, padding_below, padding_above, true),
-        op::ParameterVector{B});
-    test::Uniform<float> rng(0.0f, 1.0f);
-    vector<vector<float>> args;
-
-    for (shared_ptr<op::Parameter> param : int_f->get_parameters())
-    {
-        vector<float> tensor_val(shape_size(param->get_shape()));
-        rng.initialize(tensor_val);
-        args.push_back(tensor_val);
-    }
-    auto int_results = execute(int_f, args, "INTERPRETER");
-    auto backend_results = execute(cpu_f, args, "${BACKEND_NAME}");
-    for (size_t i = 0; i < backend_results.size(); i++)
-    {
-        EXPECT_TRUE(test::all_close(backend_results.at(i), int_results.at(i), 1.0e-4f, 1.0e-4f));
-    }
-}
+// avg_pool_3d case generation
+NGRAPH_INSTANTIATE_TEST_CASE_P(${BACKEND_NAME}, include_pad, avg_pool_3d_params, testing::Bool(), );
```

Comment:
Is the comma at the end of the line required or is it just a stray? If it is required then there needs to be some comment.